pilot/pkg/xds/debug.go

```diff
@@ -187,9 +186,6 @@ func (s *DiscoveryServer) AddDebugHandlers(mux, internalMux *http.ServeMux, enab
 		s.addDebugHandler(mux, internalMux, "/debug/force_disconnect", "Disconnects a proxy from this Pilot", s.forceDisconnect)
 	}
 
-	s.addDebugHandler(mux, internalMux, "/debug/ecdsz", "Status and debug interface for ECDS", s.ecdsz)
-	s.addDebugHandler(mux, internalMux, "/debug/edsz", "Status and debug interface for EDS", s.Edsz)
-	s.addDebugHandler(mux, internalMux, "/debug/ndsz", "Status and debug interface for NDS", s.ndsz)
```

Comment:
I think I can get behind ecds and eds, but isn't NDS completely within pilot-agent and not envoy? Just confirming that it is in fact returned in `config_dump` i.e. `istioctl pc all`

Reply:
Good point this is just pilot; I'm good with that then

Reply:
I think having /ndsz in pilot is useful.

Please add release notes as there may be folks using these endpoints in some internal tools